lib/common/ipc_client.c

```diff
@@ -1519,6 +1519,7 @@ crm_ipc_send(crm_ipc_t *client, const xmlNode *message,
 
     g_string_free(iov_buffer, TRUE);
     pcmk_free_ipc_event(iov);
+    // coverity[return_overflow]
```

Comment:
Curiosity: What's the error? An int can't overflow an int, obviously

Reply:
I think what it's actually complaining about is the various spots we assign `qb_rc` (a ssize_t) to `rc` (an int) which of course we can't really do anything about.

The actual error starts at line 1444 with:
```
1444 	        if (qb_rc <= 0) {

(38) Event cast_overflow: 	Truncation due to cast operation on "qb_rc" from 64 to 32 bits.
(39) Event overflow_assign: 	"rc" is assigned from "(int)qb_rc".
```
And then at the end:
```
1520 	    g_string_free(iov_buffer, TRUE);
1521 	    pcmk_free_ipc_event(iov);

(47) Event return_overflow: "rc", which might have overflowed, is returned from the function.
```